packages/web/src/types.ts

```diff
@@ -1,10 +1,12 @@
-import { Account as AccountSDK, LogicSigAccount } from 'algosdk';
+import { Account as AccountSDK, LogicSigAccount, Transaction } from 'algosdk';
 import * as z from 'zod';
 
 import type { ASADefSchema, ASADefsSchema } from "./types-input";
 
 export type AccountAddress = string;
 
+export type Signer = SignWithSk | SignWithLsig;
```

Comment:
this duplicates `type Sign = SignWithSk | SignWithLsig;`. Let's remove the type above.

Reply:
removed